mod-di-converter-storage-server/src/test/java/org/folio/rest/impl/JobProfileTest.java

```diff
@@ -141,7 +142,7 @@ public void shouldReturnAllProfilesOnGet() {
       .then()
       .statusCode(HttpStatus.SC_OK)
       .body("totalRecords", is(3))
-      .body("jobProfiles*.deleted", everyItem(is(false)))
+      .body("jobProfiles*.deleted", everyItem(is(nullValue())))
```

Comment:
Do we need to check for "deleted" property at all? I think we can simply remove this row. Same in other places

Reply:
Done, thank you